src/tests/scripts/split_manpage/split_manpage_tests.py

```diff
@@ -196,10 +196,11 @@ def check_completeness_of_extracted_functions_and_macros(pmdk_path, lib_path):
     macros = get_macros(pmdk_path)
     functions_from_so_files = get_functions_from_so_files(lib_path)
     functions_windows_specific = get_functions_windows_only()
+    pmem2_async_functions_group_name = ['pmem2_async']
```

Comment:
I think I'd drop `functions_` from the name of this variable - it's just a group name

Reply:
'functions_' is now dropped

Reply:
👍